### PR TITLE
Added makefile target for pytests

### PR DESCRIPTION
**Pull request by sdreyer**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -47,6 +58,7 @@ gen_grpc:						## Generates GRPC Dependencies
 
 update_python: create_venv gen_grpc 			## Updates the python package locally
 	cd test_venv/bin && source activate
+	cd ../../
```
Comment (ahmadnazeri) [inline Makefile:61]: if we are change back directories then I would recommend `source test_venv/bin/activate` on the previous line

File: Makefile [hunk 2]
```diff
@@ -22,6 +22,17 @@
 help:     						## Show this help.
 	@sed -ne '/@sed/!s/## //p' $(MAKEFILE_LIST)
 
+pytest:
+	-rm -r .featureform
```
Comment (ahmadnazeri) [inline Makefile:26]: just curious, what's the difference between `-rm` and `rm`?

**Review by ahmadnazeri — APPROVED** [review]
LGTM, just a minor suggestion.

File: Makefile (hunk 2 above)
Comment (sdreyer) [reply Makefile:26]: -rm allows it to fail, so if that directory doesn't exist, then it skips it

File: Makefile (hunk 1 above)
Comment (sdreyer) [reply Makefile:61]: Actually gonna remove this. its not the proper way to access the virtual environment with the makefile anyways

**Review by ahmadnazeri — APPROVED** [review]
LGTM

**Pull request merged**